dask/cli.py

```diff
@@ -40,6 +40,36 @@ def versions():
     show_versions()
 
 
+@cli.group()
+def config():
+    """Dask config settings"""
+    pass
+
+
+@config.command()
+@click.argument("key", default=None, required=False)
+def get(key=None):
+    """Print config key, or the whole config."""
+    from functools import reduce
+
+    from yaml import dump
+
+    from dask.config import config
```

Comment:
I assume this is to avoid eagerly loading them for every command?

Reply:
Yep, exactly.

Reply:
Sure, but I don't understand why. Certainly not for performance sake? All of which are std lib or part of mindeps so not to avoid importerror on other commands and we have imports at the top of this file anyhow. So I'm curious what makes these handled special by importing here.

Reply:
I don't think it matters. I moved them on top.

Reply:
@milesgranger: I've seen this adopted as a best practice/guideline in projects with many CLI commands where importing everything at the top slowed things down significantly (likely due to several costly imports). 

At this stage, this would be more about building a habit than actual performance improvements. That being said, until we have a consensus on such a best practice, I don't care where _cheap and required_ imports live.

Reply:
Thanks for the clarification. I've seen it with expensive imports like Keras/tensorflow and the like but wasn't aware it was also best practice for "all" imports. Forgive the noise here, but appreciate the explanation.